packages/eas-cli/src/credentials/ios/appstore/ascApiKey.ts

```diff
@@ -64,8 +72,11 @@ export async function createAscApiKeyAsync(
       }
       throw new Error(`Failed to download .p8 file of ${humanReadableKey}.`);
     }
+    // this object has more optional parameters populated
+    const fullKey = await ApiKey.infoAsync(context, { id: key.id });
+    spinner.succeed(`Created App Store Connect API Key.`);
```

Comment:
Please be consistent with `API` casing. I've seen `Api` in other files in this PR.